pkg/ffuf/progress.go

```diff
@@ -5,10 +5,11 @@ import (
 )
 
 type Progress struct {
-	StartedAt  time.Time
-	ReqCount   int
-	ReqTotal   int
-	QueuePos   int
-	QueueTotal int
-	ErrorCount int
+	StartedAt        time.Time
+	StartedAtProcess time.Time
```

Comment:
This isn't currently used anywhere in the progress output, so I think it should be removed.

Reply:
StartedAt is used in stdout.go to calculate the duration per job, so only removed StartedAtProcess.